defactor-com/ui-kit
# feat(charts): empty state working

## Empty state working

### What does this PR do?

- Resolve #152 

### Steps to test

1. Run project
2. Check changes

Now:
![imagen](https://github.com/defactor-com/ui-kit/assets/49013556/820f4b9d-2d43-423b-9852-cfacfba56cc2)
![imagen](https://github.com/defactor-com/ui-kit/assets/49013556/96caa0aa-901e-490f-9670-ffcf1b035576)
![imagen](https://github.com/defactor-com/ui-kit/assets/49013556/1f14cbce-e3f7-4f75-ad4c-4a033db8dc5d)


#### CheckList
